pkg/ddl/ddl_api.go

```diff
@@ -695,17 +695,24 @@ func (d *ddl) DropSchema(ctx sessionctx.Context, stmt *ast.DropDatabaseStmt) (er
 }
 
 func (d *ddl) RecoverSchema(ctx sessionctx.Context, recoverSchemaInfo *RecoverSchemaInfo) error {
+	involvedSchemas := []model.InvolvingSchemaInfo{{
+		Database: recoverSchemaInfo.Name.L,
+		Table:    model.InvolvingAll,
+	}}
+	if recoverSchemaInfo.OldSchemaName.L != recoverSchemaInfo.DBInfo.Name.L {
```

Comment:
```suggestion
	if recoverSchemaInfo.OldSchemaName.L != recoverSchemaInfo.Name.L {
```

I found it suprise why where are three values `recoverSchemaInfo.Name.L` `recoverSchemaInfo.DBInfo.Name.L` `recoverSchemaInfo.OldSchemaName.L` then I see it's embedded member. 😂 just avoid confusion

Reply:
unify to `recoverSchemaInfo.DBInfo.Name.L` https://github.com/pingcap/tidb/pull/54141/commits/c1ef87ffcb8c434457f7106e8c4103509f2a9607